### ISSUE TITLE
[BUG] All Current MacOS Builds Fail to Launch (MacOS 10.14.6)

### ISSUE THREAD

**Issue by Kas-tle** [solvedDev/bridge.#108]
**Summary**
All Current MacOS Builds Fail to Launch (MacOS 10.14.6)

**To Reproduce**
1. Download any of the latest release dmgs for MacOS
2. Attempt to launch bridge
3. The application quits unexpectedly

**Observed behavior**
The application quits unexpectedly
**Expected behavior**
The application should properly launch

**Screenshots / File Attachments**
A log is attached.
[crash.txt](https://github.com/solvedDev/bridge./files/3812951/crash.txt)

**Platform (please complete the following information):**
 - OS: [MacOS 10.14.6]
 - App Version: [0.13.22]

**Comment by solvedDev** [comment]
Thanks for your report! I think this was connected to bridge. using an older version of electron-builder. I would really appreciate it if you could give this another try as soon as the next version releases :)

**Comment by solvedDev** [comment]
https://github.com/solvedDev/bridge./releases/tag/v0.13.23

**Comment by Kas-tle** [comment]
Thanks for the update. The app does launch now. However, I see only a blank window. Checking the console from the inspector, I see the following error:

```
internal/validators.js:107 Uncaught TypeError [ERR_INVALID_ARG_TYPE]: The "path" argument must be of type string. Received type undefined
    at validateString (internal/validators.js:107)
    at Object.join (path.js:1037)
    at Object.<anonymous> (renderer.js:1)
    at c (renderer.js:1)
    at e (renderer.js:1)
    at Module.<anonymous> (renderer.js:7)
    at c (renderer.js:1)
    at e (renderer.js:1)
    at i (renderer.js:7)
    at renderer.js:1
```

Please let me know if there is anymore info I can provide.

**Comment by Kas-tle** [comment]
Update: Removing the debug path constant added [here](https://github.com/solvedDev/bridge./commit/bd7dcda0c3265082e76ec3880ac398900a13ae2e#diff-1fb1f50a4c01c2768c5f86a270f784da) and compiling from there seems to have fixed the issue for me. I believe it was just upset that you were trying to reference a path that would only be possible on Windows. Suppose you would have to take an approach similar to what you have [done in bridge./src/shared/Paths.js ](https://github.com/solvedDev/bridge./blob/bd7dcda0c3265082e76ec3880ac398900a13ae2e/src/shared/Paths.js). Would have done a fix myself, but I honestly don't know anything about coding. Just figured commenting out the first part would be ok since it was only added 5 days ago.

**Comment by solvedDev** [comment]
Thanks a lot! Will fix that too 👍

**Comment by Kas-tle** [comment]
Closing then, thanks so much for the help.

**Comment by solvedDev** [comment]
Will be part of the 1.0.0 release I am currently working on 👍

**Comment by solvedDev** [comment]
@Kas-tle, since I released another version before v1.0.0, [v0.13.24](https://github.com/solvedDev/bridge./releases/tag/v0.13.24) already contains the fix for MacOS :)
